frontend/testing/playwright/pages/DataModelPage.ts

```diff
@@ -72,17 +85,16 @@ export class DataModelPage extends BasePage {
     await this.page.getByRole('treeitem', { name }).focus();
   }
 
-  public async clickOnAddNodeToPropertyButton(): Promise<void> {
+  public async clickOnAddPropertyToObjectButton(propertyName: TextKey): Promise<void> {
```

Comment:
I would suggest to send in the type (e.g., number, sting, etc.) instead of using the text key in the test file. This is to keep the same structure for all playwright tests. 
```suggestiony
  public async clickOnAddPropertyToObjectButton(property:  'string' | 'number'): Promise<void> {
```

Reply:
It doesn't seem to work 😕 

<img width="765" alt="image" src="https://github.com/user-attachments/assets/2189fb84-326b-49d8-a33e-5063a8775a09">

Reply:
I can keep your suggested changes if I add a `as TextKey` in textmock?